recipe/meta.yaml

```diff
@@ -17,7 +17,8 @@ build:
     - INSTALL_PREFIX=${PREFIX}
   script:
     - cd rmm/python
-    - {{ PYTHON }} -m pip install . --no-deps -vv
+    - {{ PYTHON }} setup.py build_ext -- -DFIND_RMM_CPP=ON ${CMAKE_ARGS}
+    - {{ PYTHON }} -m pip install . --no-deps -vv ${CMAKE_ARGS}
```

Comment:
I suspect that this will fail, `pip install` won't support arbitrary options. I know that there's a `--install-option` parameter though, so it may work if you do `pip install ... --install-options="${CMAKE_ARGS}"`. If that works, we may be able to get rid of this two step process and remove the `build_ext` line.